src/containers/AddRoute/add-route.component.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable constructor-super */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-console */
+
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { successToaster, errorToaster } from '@utils';
+import { RouteService } from '@services';
+import { Route } from '../../model/index';
+import ldflex from '@solid/query-ldflex';
+import {
+  TextEditorWrapper,
+  TextEditorContainer,
+  Header,
+  Form,
+  FullGridSize,
+  Label,
+  Input,
+  TextArea,
+  WebId
+} from './add-route.style';
+
+type Props = { webId: String };
+
+export const Editor = ({ webId }: Props) => {
```

Comment:
No more editor nam. maybe RouteAddComponent?

Reply:
It's already renamed

Reply:
And according to the "error" image, that is not my code's problem since it is already happening in develop branch in which my code does not appear.

I'll open an issue to disccuss that problem